Google Play Music/Google Play Music.csproj

```diff
@@ -320,6 +320,9 @@
   </Target>
   <Import Project="..\packages\cef.redist.x86.3.2357.1287\build\cef.redist.x86.targets" Condition="Exists('..\packages\cef.redist.x86.3.2357.1287\build\cef.redist.x86.targets')" />
   <Import Project="..\packages\CefSharp.Common.43.0.0\build\CefSharp.Common.targets" Condition="Exists('..\packages\CefSharp.Common.43.0.0\build\CefSharp.Common.targets')" />
+  <PropertyGroup>
+    <PreBuildEvent>taskkill /F /IM CefSharp.BrowserSubprocess.exe 2&gt;&amp;1 | exit /B 0</PreBuildEvent>
```

Comment:
Mmmm, that makes sense, we don't want to be killing off other programs, people might not like that :smile: